{{cookiecutter.project_slug}}/clients/mobile/react-native/package.json

```diff
@@ -5,16 +5,16 @@
   "author": "Thinknmimble",
   "main": "node_modules/expo/AppEntry.js",
   "scripts": {
-    "start": "source .env && SENTRY_PROJECT_NAME=${SENTRY_PROJECT_NAME} SENTRY_AUTH_TOKEN=${SENTRY_AUTH_TOKEN} npx expo start",
-    "start:clear": "source .env && SENTRY_PROJECT_NAME=${SENTRY_PROJECT_NAME} npx expo start --clear",
-    "start:prod": "source .env && SENTRY_PROJECT_NAME=${SENTRY_PROJECT_NAME} npx expo start --no-dev --minify",
-    "start:android": "source .env && SENTRY_PROJECT_NAME=${SENTRY_PROJECT_NAME} npx expo start --android",
-    "start:ios": "source .env && SENTRY_PROJECT_NAME=${SENTRY_PROJECT_NAME} npx expo start --ios",
-    "start:web": "source .env && SENTRY_PROJECT_NAME=${SENTRY_PROJECT_NAME} npx expo start --web",
+    "start": "source .env && SENTRY_PROJECT_NAME=${SENTRY_PROJECT_NAME} SENTRY_AUTH_TOKEN=${SENTRY_AUTH_TOKEN} npx expo start --clear",
+    "start:clear": "source .env && SENTRY_PROJECT_NAME=${SENTRY_PROJECT_NAME} SENTRY_AUTH_TOKEN=${SENTRY_AUTH_TOKEN} npx expo start --clear",
+    "start:prod": "source .env && SENTRY_PROJECT_NAME=${SENTRY_PROJECT_NAME} SENTRY_AUTH_TOKEN=${SENTRY_AUTH_TOKEN} npx expo start --no-dev --minify",
+    "start:android": "source .env && SENTRY_PROJECT_NAME=${SENTRY_PROJECT_NAME} SENTRY_AUTH_TOKEN=${SENTRY_AUTH_TOKEN} npx expo start --android",
```

Comment:
added the `--clear` flag by default RN sometimes cache's the config files and this causes issues with dev's spending an hour trying to figure out why their changes didnt build.